site/styles/_shift72.scss

```diff
@@ -278,13 +282,18 @@
 
 s72-useraccount-form {
   .s72-account-password-form-group,
-  .s72-account-pin-form-group {
+  .s72-account-pin-form-group,
+  .s72-account-request-delete-form-group {
```

Comment:
Is this meant to be in the `s72-useraccount-form` block?

Reply:
Yep, its in the user account form page, the request delete button you see in the first screenshot.